### ISSUE TITLE
help shown when chaining commands does not work when there are leading spaces: 

### ISSUE THREAD

**Issue by l-ouis** [ddnet/ddnet#8619]
`command1 args; command2 args`
`              ^             `

will cause help to not be shown for command2. Removing the space shows help

**Comment by def-** [comment]
@ChillerDragon

**Comment by ChillerDragon** [comment]
> @ChillerDragon

Yep I added a feature regarding help texts. But this functionality was missing before my pr already. [And I am not planning to add it](https://github.com/ddnet/ddnet/pull/8352#issuecomment-2237927290).
